### PR TITLE
Adding SciPy 2023 blog post

### PR DESCRIPTION
**Pull request by rmshkv**
Still waiting on one tutorial recording to be uploaded, but that can be added later.

### CODE REVIEW

**Review by kafitzgerald — COMMENTED** [review]
This looks great! 🎉   Thanks for pulling this together.

A couple of minor suggestions:
* Maybe split the tutorial out into it's own category so it's more obvious which is which
* If we wanted to get this published ASAP, I think just removing the *TBD* lines would put this in a good spot where we could add items later and it also would look fine if those never appeared for whatever reason.

File: posts/2023/scipy23.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+#### Building MetPy for the Long Term: Working to Keep an Open Source Project Sustainable
+*Ryan May*
+
+- [Recording link](https://www.youtube.com/watch?v=7hbGHHaRzBA)
+- [Slides](https://docs.google.com/presentation/d/14Db0NtTWW9P9TAlO8-_Ls_r9Hotq6Jo8tjaFJMP_lfo/edit?usp=sharing)
+
+#### Tidy Geospatial Cubes
+*Emma Marshall, Deepak Cherian, Scott Henderson*
+
```
Comment (erogluorhan) [inline posts/2023/scipy23.md:45]: Checking if recording and slides are not available for this talk

**Review by erogluorhan — APPROVED** [review]
This looks great overall and approved, but please have a look at the question of mine below

File: posts/2023/scipy23.md (hunk 1 above)
Comment (rmshkv) [reply posts/2023/scipy23.md:45]: Nope, they're not - will update if they become available

**Comment by kafitzgerald** [comment]
Closes #242

**Pull request merged**